gym/lib/gym/code_signing_mapping.rb

```diff
@@ -104,6 +104,21 @@ def test_target?(build_settings)
       return (!build_settings["TEST_TARGET_NAME"].nil? || !build_settings["TEST_HOST"].nil?)
     end
 
+    def other_platform?(build_settings)
+      destination = Gym.config[:destination]
+      destination.slice! "generic/platform="
+      sdkroot = ""
+      case destination
+      when "macosx"
+        sdkroot = "macosx"
+      when "iOS"
+        sdkroot = "iphoneos"
+      when "tvOS"
+        sdkroot = "appletvos"
+      end
+      return build_settings["SDKROOT"] != sdkroot
```

Comment:
If the `SDKROOT` is `watchos` and `destination` is `iOS`, it should be handled as same platform.
The Provisioning Profiles for watchOS is necessary for archiving iOS app.